lib/Controller/SiteController.php

```diff
@@ -100,11 +100,16 @@ public function showDefaultPage() {
 		return new RedirectResponse($this->url->getAbsoluteURL('/index.php/apps/files/'));
 	}
 
-	protected function createResponse(int $id, array $site): TemplateResponse {
+	protected function createResponse(int $id, array $site, ?string $path = null): TemplateResponse {
 		$this->navigationManager->setActiveEntry('external_index' . $id);
 
+		if ($path !== null) {
```

Comment:
```suggestion
		if ($path !== '') {
```
If the parameter is empty it will just be an empty string